### PR TITLE
Add FAQ page to documentation

### PR DESCRIPTION
**Pull request by desilinguist**
- Use `sphinx_design` in docs.
- Remove outdated note from `api.rst`.
- Add references needed by FAQ page.
- Add FAQ page with first set of questions.

Happy to take suggestions of additional things that we should put in the FAQ. 

Here's the FAQ page as built by read the docs: https://rsmtool.readthedocs.io/en/547-add-faq-to-docs/faq.html

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+non-numeric feature values. No further analysis can be run.`` **What
+happened?** [:ref:`# <q_no_response_remaining>`]
+
+``rsmtool`` is designed to work only with numeric features. Non-numeric values
+*including* missing values are filtered out. Some of the common reasons for this
+error are:
+
+- The human score column or one of the features contains only non-numeric
+  values. You can either exclude this feature or convert it to one-hot
+  encoding.
```
Comment (tamarl08) [inline doc/faq.rst:55]: add a reference link to one-hot encoding? wiki page or something better if you have.

File: doc/faq.rst [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+:ref:`transformations<select_transformations_rsmtool>` to some of the features.
+Some transformations such as ``inv`` (inverse transform) change the polarity of
+the feature.  In such cases, RSMTool takes this into account and changes the
+sign accordingly. See :ref:`note here<clever_sign_note>`.
+
+|hr|
+
+.. _q_mismatched_rsmpredict_rsmtool:
+
+:fa:`quora`. I ran ``rsmpredict`` to generate predicted scores using a model
```
Comment (tamarl08) [inline doc/faq.rst:109]: This question font is not bold

Comment (damien2012eng) [inline doc/faq.rst:109]: Looks like the rest of questions are bolded. Should this one be bolded too?

File: doc/faq.rst [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+trained with ``rsmtool``. I used the evaluation set from ``rsmtool`` as the input
+data for ``rsmpredict``. I expected the number of excluded/included responses
+to be the same. However, for some reason, ``rsmpredict`` generated the
+predicted scores for more cases than ``rsmtool``.
+[:ref:`# <q_mismatched_rsmpredict_rsmtool>`]
+
+The situation described here often occurs in the following situation.
```
Comment (damien2012eng) [inline doc/faq.rst:116]:
```suggestion
The situation described here often occurs in the following situations.
```

File: doc/faq.rst (hunk 2 above)
Comment (tamarl08) [reply doc/faq.rst:109]: also, suggesting to make the question shorter, something like:
rsmpredict generates a different number of results than rsmtool, on the same evaluation set.

**Review by damien2012eng — APPROVED** [review]
LGTM! All links are tested.

File: doc/faq.rst [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+As a result, there is a mismatch in the number of predictions generated by
+``rsmtool`` and ``rsmpredict``.
+
+|hr|
+
+.. _q_relative_betas_not_one:
+
+:fa:`quora`. **The relative betas for the linear regression model in my**
+``rsmtool`` **run do not sum to 1. Is that a problem?** [:ref:`#
```
Comment (tamarl08) [inline doc/faq.rst:138]:
```suggestion
``rsmtool`` **run do not sum up to 1. Is that a problem?** [:ref:`#
```

File: doc/faq.rst [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+<q_relative_betas_not_one>`]
+
+Please check if your model has any negative coefficients. Relative coefficients
+*only* make sense when all model coefficients are positive. Their sum is
+expected to be less than 1 if there are negative coefficients. Note that if
+this is the case the relative coefficients will *not* be included into the report.
```
Comment (tamarl08) [inline doc/faq.rst:144]:
```suggestion
this is the case, the relative coefficients will *not* be included into the report.
```

File: doc/faq.rst [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+.. _q_wandb:
+
+:fa:`quora`. **Can I use Weights & Biases to log artifacts and reports?**
+[:ref:`# <q_wandb>`]
+
+Yes! W&B logging is natively supported the following command-line tools:
```
Comment (tamarl08) [inline doc/faq.rst:151]:
```suggestion
Yes! W&B logging is natively supported in the following command-line tools:
```

File: doc/faq.rst (hunk 4 above)
Comment (desilinguist) [reply doc/faq.rst:138]: I am not sure this is really necessary in American English :)

**Review by tamarl08 — APPROVED** [review]

**Pull request merged**